src/Assets/TestProjects/DesktopAppWithLibrary-PackagesConfig/TestApp/TestApp.csproj

```diff
@@ -78,8 +78,8 @@
     <Reference Include="System.Security.Cryptography.Encoding, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>$(NUGET_PACKAGES)\System.Security.Cryptography.Encoding.4.3.0\lib\net46\System.Security.Cryptography.Encoding.dll</HintPath>
     </Reference>
-    <Reference Include="System.Security.Cryptography.Primitives, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>$(NUGET_PACKAGES)\System.Security.Cryptography.Primitives.4.3.0\lib\net46\System.Security.Cryptography.Primitives.dll</HintPath>
+    <Reference Include="System.Security.Cryptography, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>$(NUGET_PACKAGES)\System.Security.Cryptography.4.3.0\lib\net46\System.Security.Cryptography.dll</HintPath>
```

Comment:
There is no package for System.Security.Cryptography (and there hasn't been one for Primitives for a while); it's just part of the shared runtime (and FWIW its lowest assembly version is 7.0.0.0)